logstash-core/lib/logstash/config/defaults.rb

```diff
@@ -6,6 +6,14 @@ module LogStash module Config module Defaults
 
   extend self
 
+  def input
+    "input { stdin { type => stdin } }"
+  end
+
+  def output
+    "output { stdout { codec => rubydebug } }"
```

Comment:
Hmmm, I got that from here: https://github.com/elastic/logstash/blob/00a99c19e512a6ec0089cf5996168cbc15470330/logstash-core/lib/logstash/runner.rb#L18 

Is that incorrect?